R/automateDataPreparation.R

```diff
@@ -7,6 +7,22 @@ automateDataPreparation <- function(datList = NULL, inputList, path = NULL,
 						addLeadingZeros=FALSE, truncateSpaceChar = TRUE, newID = NULL, oldIDs = NULL,
             missing.rule = list(mvi = 0, mnr = 0, mci = NA, mbd = NA, mir = 0, mbi = 0), verbose=FALSE) {
 
+  checkmate::assert_list(datList, types = "data.frame", null.ok = TRUE)
```

Comment:
this check is too restrictive. It should be allowed that datList is just a single data.frame. When I do this I get an error:
Fehler: Assertion on 'singeDat' failed: Must be of type 'list' (or 'NULL'), not 'data.frame'. I added a test for this scenario that must pass.